heartbeat/_meta/fields.common.yml

```diff
@@ -196,46 +203,64 @@
           fields:
             - name: experience
               type: group
+              description: >
+                Absolute values of all user experience metrics in the browser
+                relative to the navigation start event in microseconds
               fields:
-                - name: name
-                  type: keyword
-                - name: type
-                  type: text
-                  description: >
-                    denotes the 'mark' event
-                - name: start
-                  type: long
-                  description: >
-                    offset of time relative to journey start in milliseconds
-            - name: user_timing
+                - name: fcp
+                  type: group
+                  description: duration of First contentful paint metric
+                  fields:
+                    - name: us
+                      type: integer
+                - name: lcp
+                  type: group
+                  description: duration of Largest contentful paint metric
+                  fields:
+                    - name: us
+                      type: integer
+                - name: dcl
+                  type: group
+                  description: duration of Document content loaded end event
+                  fields:
+                    - name: us
+                      type: integer
+                - name: load
+                  type: group
+                  description: duration of Load end event
+                  fields:
+                    - name: duration
+                      type: integer
+                - name: cls
+                  type: integer
+                  description: culumative layout shift score across all frames
+            - name: relative_trace
               type: group
+              description: >
+                trace event with timing information that are realtive to
+                journey timings in microseconds
               fields:
                 - name: name
                   type: keyword
+                  description: name of the trace event
                 - name: type
                   type: text
-                  description: >
-                    could be one of mark or measure event types.
+                  description: could be one of mark or measure event types
                 - name: start
-                  type: long
-                  description: >
-                    offset of time relative to journey start in milliseconds
+                  type: group
+                  description: monotonically increasing trace start time in microseconds
+                  fields:
+                    - name: us
+                      type: long
                 - name: end
```

Comment:
We should use a start plus duration instead of a start and an end time in hindsight. Playing around with this in kibana it's hard to create a dashboard of how long the event took with just these two values because you need to use subtraction to derive the duration.

Alternatively, we could keep the end and make the duration and additive change. Can you think of a reason why someone might want to perform a query by the end time of an event? That might make sense to me

Reply:
Yeah good point, Lets add the `duration.us` to relative trace although its applicable only for `measure` event types as `mark` event types are meant to be markers in the network timing and not a duration metric.

Reply:
Done.